cunumeric/array.py

```diff
@@ -3285,14 +3442,12 @@ def setflags(
         # Be a bit more careful here, and only pass params that are explicitly
         # set by the caller. The numpy interface specifies only bool values,
         # despite its None defaults.
-        kws = {}
         if write is not None:
-            kws["write"] = write
+            self._flags["WRITEABLE"] = write
```

Comment:
now I see why you needed to deviate from NumPy's flags... can't you just use the shorthand names here?

Reply:
379c90dbfd42e8b35fdf1f0e3237d8a2ef2a1e12 but I will probably try to clean these up more separately.